models/src/main/scala/it/pagopa/interop/agreementmanagement/model/persistence/AgreementEventsSerde.scala

```diff
@@ -11,6 +11,8 @@ object AgreementEventsSerde {
     case x: AgreementConsumerDocumentAdded   => x.toJson
     case x: AgreementConsumerDocumentRemoved => x.toJson
     case x: AgreementAdded                   => x.toJson
+    case x: AgreementDeleted                 => x.toJson
+    case x: AgreementSubscribed              => x.toJson
```

Comment:
"Submitted" makes more sense to me than "Subscribed" in this scenario

Reply:
You're right.